rootfs/standard/var/www/mynode/settings.py

```diff
@@ -121,6 +120,45 @@ def upgrade_device():
     # Reboot
     reboot_device()
 
+def read_ui_settings():
+    ui_hdd_file = '/mnt/hdd/mynode/settings/ui.json'
+    ui_mynode_file = '/home/bitcoin/.mynode/ui.json'
+
+    # read ui.json from HDD
+    if os.path.isfile(ui_hdd_file):
+        with open(ui_hdd_file, 'r') as fp:
+            ui_settings = json.load(fp)
+    # read ui.json from mynode
+    elif os.path.isfile(ui_mynode_file):
+        with open(ui_mynode_file, 'r') as fp:
+            ui_settings = json.load(fp)
+    # if ui.json is not found anywhere, use default settings
+    else:
+        ui_settings = {'darkmode': False}
+
+    return ui_settings
+
+def write_ui_settings(ui_settings):
```

Comment:
You might want to wrap this in a try since the HDD write will fail if the setting is changed before an HDD is detected.